common/test/acceptance/fixtures/catalog.py

```diff
@@ -40,16 +40,16 @@ def install_programs(self, programs):
             data={key: json.dumps(uuids)},
         )
 
-    def install_credit_pathways(self, credit_pathways):
+    def install_pathways(self, pathways):
         """
         Stub the discovery service's credit pathways API endpoint
 
         Arguments:
-             credit_pathways (list): A list of credit pathways. List endpoint will be stubbed using data from this list.
+             pathways (list): A list of credit pathways. List endpoint will be stubbed using data from this list.
```

Comment:
Nit: Update the description of the pathways arg

Reply:
I'm fine with postponing this fix until we come back in for the professional pathways change.  Don't want to rerun 45 minutes of tests just for a comment fix